project/Dependencies.scala

```diff
@@ -46,6 +46,7 @@ object Dependencies {
 
     val sttp = "1.7.2"
```

Comment:
Also please remove this, it is not used anymore )

Reply:
resolved